understanding/focus-order.html

```diff
@@ -58,205 +56,151 @@
-                     if the focus order is not logical.
+                  <li>画面拡大ソフトを使用していて、拡大率を高くしている利用者には、ページのごく小さい一部分だけしか見えないことがある。そのような利用者は、フォーカス順序が論理的でないと、誤った文脈でコンテンツの一部分を解釈してしまう恐れがある。
                      
                   </li>
                   
                   
                </ul>
             </section>
             <section id="examples">
-               <h2>Examples</h2>
+               <h2>事例</h2>
                <ul>
                   
                   
-                  <li>On a web page that contains a tree of interactive controls, the user can use the up
-                     and down arrow keys to move from tree node to tree node. Pressing the right arrow
-                     key expands a node, then using the down arrow key moves into the newly expanded nodes.
+                  <li>インタラクティブなコントロールのツリーがあるウェブページ上で、利用者は上下の矢印キーを使って、ツリーのノードからノードへと移動することができる。右向き矢印キーを押下するとノードを展開して、それから下矢印キーを使用すると新しく展開されたノードへと移動していく。
                      
                   </li>
                   
                   
-                  <li>A Web page implements modeless dialogs via scripting. When the trigger button is activated,
-                     a dialog opens. The interactive elements in the dialog are inserted in the focus order
-                     immediately after the button. When the dialog is open, the focus order goes from the
-                     button to the elements of the dialog, then to the interactive element following the
-                     button. When the dialog is closed, the focus order goes from the button to the following
-                     element.
-                     
+                  <li>あるウェブページは、スクリプトでモードレスダイアログを実装している。起動ボタンを動作させると、ダイアログが開く。ダイアログ内にあるインタラクティブな要素が、起動ボタンのすぐ後のフォーカス順序の位置に挿入される。ダイアログが開いているときには、フォーカス順序は、その起動ボタンからダイアログ内の要素へ移動し、それから起動ボタンの後にあるインタラクティブな要素へと移動する。ダイアログが閉じているときは、フォーカス順序は起動ボタンからその次に続いている要素へ移動する。
+                  <div class="note"><div role="heading" class="note-title marker" aria-level="3">訳注</div><p>モードレスダイアログとは、ダイアログボックスを閉じなくても作業が続行できるタイプのダイアログボックスのことである。<a href="https://e-words.jp/w/%E3%83%A2%E3%83%BC%E3%83%89%E3%83%AC%E3%82%B9%E3%83%80%E3%82%A4%E3%82%A2%E3%83%AD%E3%82%B0.html">モードレスダイアログ（モードレスウィンドウ）とは - IT 用語辞典</a>も参照のこと。</p></div>
                   </li>
                   
                   
-                  <li>A Web page implements modal dialogs via scripting. When the trigger button is activated,
-                     a dialog opens and focus is set within the dialog. As
-                     long as the dialog is open, focus is limited to the elements of the dialog. When the
-                     dialog is dismissed, focus returns to the button or the element following the button.
-                     
+                  <li>あるウェブページは、スクリプトでモーダルダイアログを実装している。起動ボタンを動作させると、ダイアログが開き、そのダイアログの中にフォーカスがあたる。ダイアログが開いている限り、フォーカスはダイアログ内の要素だけに限定される。ダイアログが閉じられると、フォーカスはボタン又はボタンの次にある要素へ戻る。
```

Comment:
常用漢字表にある漢字を使います。
https://www.bunka.go.jp/seisaku/bunkashingikai/kokugo/hokoku/pdf/93651301_01.pdf#page=20

```suggestion
                  <li>あるウェブページは、スクリプトでモーダルダイアログを実装している。起動ボタンを動作させると、ダイアログが開き、そのダイアログの中にフォーカスが当たる。ダイアログが開いている限り、フォーカスはダイアログ内の要素だけに限定される。ダイアログが閉じられると、フォーカスはボタン又はボタンの次にある要素へ戻る。
```